src/plugins/service-worker/index.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+const buildPaths = require('mwp-cli/src/commands/buildCommands/config/paths');
```

Comment:
I'd like to take a look at what can be done here, even if its just showing me all the hazards of going down this path. 

Interdependencies can get complicated very quickly as well.

I'll create a follow up ticket.